### ISSUE TITLE
`priorsigmafac` should be of length `factors`

### ISSUE THREAD

**Issue by hdarjus** [gregorkastner/factorstochvol#8]
https://github.com/gregorkastner/factorstochvol/blob/059dc4774ab0792baa81d0c251d6905fafb391a4/R/wrappers.R#L559

@2malK reports that this line seems incorrect. It should be `priorsigmafac <- rep(priorsigmafac, factors)`. However, things run smoothly even for bad input:

```r
set.seed(23)
sim_res <- fsvsim(n = 100, series = 3, factors = 2)
fsvsample(y = sim_res$y, factors = 2,
          priorsigmafac = rep(1, 2))
```

**Comment by hdarjus** [comment]
correction: there is error for correct input. See example.

**Comment by hdarjus** [comment]
solved in commits c3f2614d6942ea4590d9956022d4118ff82a9510 and eb34cec04a0290d6bfbc0a5876c5c18fc61ae22b
